build/patches/Enable-native-Android-autofill.patch

```diff
@@ -11,45 +11,50 @@ along with browser-managed autofill.
+ android_webview/browser/aw_contents.cc        |  2 +-
  chrome/android/BUILD.gn                       |  1 +
- .../settings/PasswordSettings.java            | 81 ++++++++++++++++++-
- .../chromium/chrome/browser/tab/TabImpl.java  | 50 ++++++++++++
+ .../settings/PasswordSettings.java            | 82 ++++++++++++++++++-
+ .../chromium/chrome/browser/tab/TabImpl.java  | 50 +++++++++++
  .../browser/tab/TabViewAndroidDelegate.java   | 14 ++++
  chrome/browser/BUILD.gn                       |  8 ++
- chrome/browser/android/tab_android.cc         | 27 +++++++
+ chrome/browser/android/tab_android.cc         | 23 ++++++
  chrome/browser/android/tab_android.h          |  2 +
  .../tab_web_contents_delegate_android.cc      |  3 +-
  .../strings/android_chrome_strings.grd        |  6 ++
- chrome/browser/ui/tab_helpers.cc              |  8 +-
- .../browser/content_autofill_driver.cc        | 50 ++++++++++--
- .../content/browser/content_autofill_driver.h |  8 +-
- .../content_autofill_driver_factory.cc        |  9 ++-
- .../browser/content_autofill_driver_factory.h |  4 +
+ chrome/browser/ui/tab_helpers.cc              |  6 +-
+ .../browser/android_autofill_manager.cc       |  2 +-
+ .../browser/android_autofill_manager.h        | 10 +--
+ .../browser/content_autofill_driver.cc        | 29 +++++++
+ .../content/browser/content_autofill_driver.h |  9 +-
+ .../content_autofill_driver_factory.cc        |  7 +-
+ .../browser/content_autofill_driver_factory.h |  1 +
  .../renderer/password_autofill_agent.cc       |  5 +-
  .../autofill/core/common/autofill_prefs.cc    |  8 ++
  .../autofill/core/common/autofill_prefs.h     |  2 +
  .../embedder_support/view/ContentView.java    | 46 +++++++++++
  .../chromium/ui/base/ViewAndroidDelegate.java |  8 ++
- weblayer/browser/tab_impl.cc                  |  3 +-
- 21 files changed, 329 insertions(+), 15 deletions(-)
+ 22 files changed, 311 insertions(+), 13 deletions(-)
 
 diff --git a/android_webview/browser/aw_contents.cc b/android_webview/browser/aw_contents.cc
 --- a/android_webview/browser/aw_contents.cc
 +++ b/android_webview/browser/aw_contents.cc
-@@ -335,6 +335,7 @@ void AwContents::InitAutofillIfNecessary(bool autocomplete_enabled) {
-       autofill::AutofillProvider::is_download_manager_disabled_for_testing()
-           ? autofill::AutofillManager::DISABLE_AUTOFILL_DOWNLOAD_MANAGER
-           : autofill::AutofillManager::ENABLE_AUTOFILL_DOWNLOAD_MANAGER,
-+      /*enable_browser_autofill_manager*/ false,
+@@ -339,7 +339,7 @@ void AwContents::InitAutofillIfNecessary(bool autocomplete_enabled) {
        autofill_provider
-           ? base::BindRepeating(&autofill::AndroidAutofillManager::Create)
-           : autofill::AutofillManager::AutofillManagerFactoryCallback());
+           ? base::BindRepeating(&autofill::AndroidDriverInitHook,
+                                 AwAutofillClient::FromWebContents(web_contents),
+-                                enable_download_manager)
```

Comment:
I did it like this

```
diff --git a/android_webview/browser/aw_contents.cc b/android_webview/browser/aw_contents.cc
--- a/android_webview/browser/aw_contents.cc
+++ b/android_webview/browser/aw_contents.cc
@@ -336,6 +336,7 @@ void AwContents::InitAutofillIfNecessary(bool autocomplete_enabled) {
   // WebView browser tests may shall use BrowserAutofillManager if
   // `!autofill_provider`.
   ContentAutofillDriverFactory::DriverInitCallback driver_init_hook =
+      /*enable_browser_autofill_manager*/ false,
       autofill_provider
           ? base::BindRepeating(&autofill::AndroidDriverInitHook,
                                 AwAutofillClient::FromWebContents(web_contents),
```
however, I have not actually compiled the webview

Reply:
Yes, the `/*enable_browser_autofill_manager*/ false,` is out of place.